Dockerfile

```
@@ -0,0 +1,107 @@
+FROM ubuntu:20.04
+
+LABEL "name"="Statick"
+LABEL "version"="0.0"
+LABEL "repository"="https://github.com/sscpac/statick-action.git"
+LABEL "homepage"="https://github.com/sscpac/statick-action"
+LABEL "maintainer"="Thomas Denewiler <tdenewiler@gmail.com>"
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+USER root
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    build-essential \
+    cccc \
+    chktex \
+    clang \
+    clang-6.0 \
+    clang-10 \
+    clang-format \
+    clang-format-6.0 \
+    clang-format-10 \
+    clang-tidy \
+    clang-tidy-6.0 \
+    clang-tidy-10 \
+    cmake \
+    cppcheck \
+    curl \
+    file \
+    findbugs \
+    flawfinder \
+    git \
+    lacheck \
+    libomp-dev \
+    libpcre3-dev \
+    libperl-critic-perl \
+    libxml2 \
+    libxml2-utils \
+    maven \
+    pylint \
+    python3-pip \
+    python3-pylint-django \
+    python3-yaml \
+    python3-yapsy \
+    shellcheck \
+    uncrustify \
+    wget
+RUN python3 -m pip install --upgrade pip
+RUN python3 -m pip install --upgrade \
+    bandit \
+    black \
+    cmakelint \
+    cpplint \
+    docformatter \
+    flake8 \
+    flake8-blind-except \
+    flake8-builtins \
+    flake8-class-newline \
+    flake8-comprehensions \
+    flake8-docstrings \
+    flake8-import-order \
+    flake8-quotes \
+    flawfinder \
+    isort \
+    lizard \
+    mypy \
+    pycodestyle \
+    pydocstyle \
+    pyflakes \
+    pylint \
+    setuptools \
+    statick \
+    statick-md \
```

```suggestion
    statick-md \
    statick-planning \
```

That package is available and would be nice to have. Could be done in a separate issue later.

---

Will handle in the PR that moves the Dockerfile out of this repo.